paasta_tools/spark_tools.py

```diff
@@ -184,8 +184,10 @@ def auto_add_timeout_for_spark_job(
     if "spark-submit" not in cmd:
         return cmd
     try:
+        options_regex = "((-[a-zA-Z]+|\\-\\-[a-zA-Z\\-]+)(\\s+[a-zA-Z0-9\\-]+)?\\s*)*"
```

Comment:
```suggestion
        options_regex = r"(--*[a-zA-Z]+(\s+\w+|=\w+)?\s*)*"
```
to cover `--kill-after=1m`